### PR TITLE
add city cell

### PR DESCRIPTION
**Pull request by iRohitGaur**
added city cell with following functionalities
- 1
- 2
- 3

### CODE REVIEW

File: TableViewApp/TableViewApp/CityCell.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+//
+
+import UIKit
+
+class CityCell: UITableViewCell {
+  
+  private let titleLabel:UILabel = {
+    let label = UILabel()
+    label.text = ""
+    label.textAlignment = .center
```
Comment (iRohitGaur) [inline TableViewApp/TableViewApp/CityCell.swift:15]: suggestion: title label alignment will look better in left side

File: TableViewApp/TableViewApp/CityCell.swift [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+//
+//  Created by RG on 08/04/22.
+//
+
+import UIKit
+
+class CityCell: UITableViewCell {
+  
+  private let titleLabel:UILabel = {
+    let label = UILabel()
```
Comment (vkaushikp) [inline TableViewApp/TableViewApp/CityCell.swift:13]: suggestion: add image in city cell

File: TableViewApp/TableViewApp/CityCell.swift [hunk 3]
```diff
@@ -0,0 +1,39 @@
+//
+//  CityCell.swift
+//  TableViewApp
+//
+//  Created by RG on 08/04/22.
+//
+
+import UIKit
```
Comment (bhikadiyakenildcs23) [inline TableViewApp/TableViewApp/CityCell.swift:8]: okay!

File: TableViewApp/TableViewApp/CityCell.swift [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+class CityCell: UITableViewCell {
```
Comment (Randomnilay) [inline TableViewApp/TableViewApp/CityCell.swift:10]: noobs have no right to suggest

File: TableViewApp/TableViewApp/CityCell.swift [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    label.textAlignment = .center
+    return label
+  }()
+  
+  private let descriptionLabel:UILabel = {
+    let label = UILabel()
+    label.text = ""
+    label.textAlignment = .center
+    return label
+  }()
```
Comment (iRohitGaur) [inline TableViewApp/TableViewApp/CityCell.swift:24]: appreciation: this is a nice way to create this element

Comment (vkaushikp) [inline TableViewApp/TableViewApp/CityCell.swift:24]: this one is good

**Review by bhikadiyakenildcs23 — APPROVED** [review]
That's Awesome...!

**Review by vkaushikp — APPROVED** [review]
👍

File: TableViewApp/TableViewApp/CityCell.swift [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  private let titleLabel:UILabel = {
+    let label = UILabel()
+    label.text = ""
+    label.textAlignment = .center
+    return label
+  }()
+  
+  private let descriptionLabel:UILabel = {
+    let label = UILabel()
+    label.text = ""
```
Comment (vaishnavigorakhiya) [inline TableViewApp/TableViewApp/CityCell.swift:21]: ohky

**Review by vaishnavigorakhiya — APPROVED** [review]

File: TableViewApp/TableViewApp/CityCell.swift [hunk 7]
```diff
@@ -36,4 +36,6 @@ class CityCell: UITableViewCell {
     
     setupUI()
   }
+
+//aditional comment
 }
```
Comment (vaishnavigorakhiya) [inline TableViewApp/TableViewApp/CityCell.swift:41]: nice

**Pull request merged**